### ISSUE TITLE
Problem with tables

### ISSUE THREAD

**Issue by colomet** [pressbooks/pressbooks#524]
### Prerequisites

* [ yes] Can you reproduce the behavior you wish to address with [all other plugins deactivated](http://codex.wordpress.org/Multisite_Network_Administration#Plugins)?
* [yes ] Are you running the [latest version of Pressbooks](http://github.com/pressbooks/pressbooks/releases/latest/) and the [latest version of WordPress](http://codex.wordpress.org/Upgrading_WordPress)?
* [ no] Are you experiencing this issue with one of our built-in root or book themes?
* [ yes] Did you [perform a cursory search](https://github.com/issues?q=+is%3Aissue+repo%3Apressbooks%2Fpressbooks) to see if your bug or enhancement is already listed?

### Description

[A new editor do not desapear once you do not need any more]

### Steps to Reproduce

1. [Create a table]
2. [Press twice inside of one cell]
3. [If you press out of the cell, disappear]
4. If you press again once in the table, you have it again.

**Expected behavior:** [The new editor disappear and let you work once you do not need it]

**Actual behavior:** [The new editor is fix in the same place, is hard to write data if the ned editor is on top of the cell i need to change]

![captura de pantalla 2016-11-02 a las 18 14 32](https://cloud.githubusercontent.com/assets/3929177/19939449/56c407a8-a129-11e6-914f-bbd9ccd9dd03.png)

### Themes

Root Theme: ["Pressbooks Publisher"]
Book Theme: [Child theme of "Luther"]

### Versions

Pressbooks: [Pressbooks 3.9.3]
WordPress: [4.6.1]
PHP: [7.0.12 ]

You can get the version of Pressbooks from the readme.txt file in the Pressbooks plugin (usually located at `http://<yourdomain.tld>/wp-content/plugins/pressbooks/readme.txt`) and the version of WordPress from the readme.html file in the WordPress root directory (usually located at `http://<yourdomain.tld>/readme.html`). You can usually get your PHP version from your web host, or by using the [`phpversion()`](http://php.net/manual/en/function.phpversion.php) function.

**Comment by thomasdumm** [comment]
I think it is a positioning issue when comparing it with the demo page on the tinymce website.

**Comment by greatislander** [comment]
@colomet @thomasdumm It's funny because I am using the latest TinyMCE distribution via Bower:

https://github.com/pressbooks/pressbooks/blob/dev/bower.json#L33
https://github.com/pressbooks/pressbooks/blob/dev/assets/src/manifest.json#L183-L187
https://github.com/tinymce/tinymce-dist/releases/tag/4.4.3

I can only assume that there is some conflict between the WordPress TinyMCE variant and the core distribution. I am investigating, thanks for your patience.

**Comment by greatislander** [comment]
I looked into the table editor bundled with [TinyMCE Advanced](https://wordpress.org/plugins/tinymce-advanced) and noted that the developers of that plugin have disabled the inline toolbar (see the line `$init['table_toolbar'] = false;` [here](https://plugins.svn.wordpress.org/tinymce-advanced/tags/4.4.1/tinymce-advanced.php)). I tried enabling the inline toolbar within TinyMCE Addvanced on a vanilla WordPress install, and encountered the same issue. It looks like some of WordPress' custom TinyMCE code breaks this inline toolbar, so I have applied the same setting from TinyMCE to disable it in Pressbooks as well. An update with this fix is forthcoming.
